interface/src/app/analyze/analyze.js

```diff
@@ -58,9 +54,7 @@ angular.module('adage.analyze', [
       },
 
       scroll_to_id: function(id) {
-        $log.info('scroll_to_id called with: ' + id);
```

Comment:
Right. It had been left in there from when I was initially building the feature. There is no reason for it to stay... it was just for debugging.